tools/relay/README.md

```diff
@@ -0,0 +1,47 @@
+# Relay
+
+Relay will act as an interface between invoker and the benchmark server, allowing the user of a function to supply a custom form of input than a strict Helloworld format (see `aes.proto` for example).
+
+## Adding relay
+
+For docker compose files, add the following snippet
+
+```yaml
+relay:
+  image: vhiveease/relay:latest
+  entrypoint:
+    - /app/server
+    - --addr=0.0.0.0:50000
+    - --function-endpoint-url=${FUNCTION_URL}
```

Comment:
In general people usually use <> when they want to make part to be replaced or not?
$ chould be interpreted as environmental variable and I dont know if that works and is what you want to show...?